kusto/query_builder.go

```diff
@@ -506,6 +507,143 @@ func NewStmt(query stringConstant, options ...StmtOption) Stmt {
 	return s
 }
 
+func (s Stmt) AddDatabase(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) AddTable(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) AddColumn(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) AddFunction(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) SafeAddName(query string) Stmt {
+	if !(strings.Contains(query, "'")) {
+		query = "['" + query + "']"
+	}
+
+	if !(query == "" || strings.HasPrefix(query, "[")) {
+		query = "[\"" + query + "\"]"
+	}
+
+	s.queryStr = s.queryStr + query
+	return s
+}
+
+func (s Stmt) AddQuotedString(value string, hidden bool) Stmt {
+	if value == "" {
+		return s
+	}
+
+	literal := ""
```

Comment:
https://www.educative.io/answers/how-to-use-string-builder-in-golang

Reply:
Didn't you say in the mail not to use a string builder? thats why i avoided it

Reply:
Sorry for the confusion. I was not aware of Go having a string builder class and meant this was not a requirement.
Since it exists, please use it :)